### PR TITLE
Update readme.md

### LINKED ISSUE #127: DOC: Update README.md
- Need to update README.md file with latest images from GitOpener website.
- Add images by drag & drop 
- Add emojis
- Add some descriptions about Contributors, LICENSE etc.

### PR DESCRIPTION
**Pull request by AshwanthramKL**
## Fixes Issue

Closes #127 

## Changes proposed

- Added Emojis to Readme.md
- Added description about MIT License to Readme.md
- Added a Table of Contents to Readme.md
- Updated images of website in Readme.md
## Note to reviewers

Upon research, I found that once the PR is merged the images will be added to the user profile and the links to the images will work and not be broken.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,87 @@
-# GitOpener
+# GitOpener 🚪🔓
 
 [![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/pbclife/gitopener.vercel.app)
 
-![landingPage](https://user-images.githubusercontent.com/95094057/218668398-2e6bbe3c-c4d1-41c4-a1f3-184c94836ac4.png)
+![landingPage](https://user-images.githubusercontent.com/95094057/gitopener.vercel.app/public/images/readme/landingPage.png)
```
Comment (sboy99) [inline README.md:5]: just drag & drop the images into github editor and github will automatically upload those files and then reference those images. Currently, it's not loading the images, do this for all images

**Review by sboy99 — CHANGES_REQUESTED** [review]
Hey @AshwanthramKL , its looking really cool. Just a few fixes, as images are not rendering.

File: README.md (hunk 1 above)
Comment (AshwanthramKL) [reply README.md]: Check now

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Review by sboy99 — APPROVED** [review]
Awesome

**Pull request merged**